pkg/certificates/certificates.go

```diff
@@ -0,0 +1,24 @@
+package certificates
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/openshift/cluster-logging-operator/pkg/logger"
+)
+
+func GenerateCertificates(namespace, scriptsDir, logStoreName, workDir string) (err error) {
+	script := fmt.Sprintf("%s/cert_generation.sh", scriptsDir)
+	return RunCertificatesScript(namespace, logStoreName, workDir, script)
+}
+
+func RunCertificatesScript(namespace, logStoreName, workDir, script string) (err error) {
```

Comment:
> Why do we need to run a script to gen certs? In the interest of reducing our bash madness, can we do this with https://golang.org/pkg/crypto/x509/ ? Just asking, I haven't thought it thru.

We can, we have an open JIRA to replace the script (which is log time legacy) with go code but it hasn't been a priority since it worked and is tech debt

Reply:
https://issues.redhat.com/browse/LOG-422